bin/run-database-migrations.sh

```diff
@@ -38,15 +38,17 @@ fi
 
 DB_MIGRATOR_USER=$(terraform -chdir=infra/$APP_NAME/app-config output -json environment_configs | jq -r ".$ENVIRONMENT.database_config.migrator_username")
 
+echo "::endgroup::"
```

Comment:
since step 0 doesn't print anything in the case where there's a database, and even if there's no database it only prints one line then exits, i think we should leave Step 0 ungrouped